### ISSUE TITLE
Volumn with Plain text names is not supported

### ISSUE THREAD

**Issue by velu4ever** [hardcore-sushi/DroidFS#256]
When the volume is created with an option of using plain text for filenames as opposed to encrypted filenames, an additional file "gocryptfs.diriv" is not created in every folder and subfolder to track the name encryption. 
Due to this, When I mount and try to open one such volume in DroidFS, it displays a configuration error message. 
I am using the unencrypted filenames to easily identify the folders I need to work on so that I can separately mount them to get faster read/write performance on PC.
I am using the latest version at the time of writing (v2.1.2) on android 14 poco f5

**Comment by velu4ever** [comment]
First of all, I must say I am so grateful to you for bringing this app to android.
PFA for the volume zip file and Screenshot that shows the error and version. I used the latest version in hopes of not getting the error. As it was still there, I opened this issue. I have given the steps I followed yesterday to get this error.

Steps:
1) create a volume in PC with plain_text for filename. Other settings can remain the same.
2) mount the volume in PC
3) add a folder and make a note of the folder name
4) add some files and make note of those names as well
5) dismount the volume
6) go to the cipher volume and check the filenames with the note.
7) now, copy the cipher folder to android phone
8) try mounting the volume

I got the error and shared the screenshot.
PW: asdf1234
Ty

![Screenshot](https://github.com/hardcore-sushi/DroidFS/assets/56683234/e6cc8a3c-c9d6-401a-97d3-76033fc0b140)
[cipher_v.zip](https://github.com/hardcore-sushi/DroidFS/files/13608230/cipher_v.zip)
